e2e-tests/ukef/cypress/e2e/journeys/portal/deal-cancellation/BSS-EWCS/AIN-cancellation-updates-portal.spec.js

```diff
@@ -0,0 +1,85 @@
+import portalPages from '../../../../../../../portal/cypress/e2e/pages';
+import MOCK_USERS from '../../../../../../../e2e-fixtures/portal-users.fixture';
+import generateAinReadyToSubmit from '../../test-data/AIN-deal/dealReadyToSubmit';
+import { TFM_URL, PIM_USER_1 } from '../../../../../../../e2e-fixtures';
+import { yesterday } from '../../../../../../../e2e-fixtures/dateConstants';
+import { submitDealCancellation } from '../../../../../support/trade-finance-manager-ui/submit-deal-cancellation';
+
+const { BANK1_MAKER1, BANK1_CHECKER1 } = MOCK_USERS;
+
+describe('Deal Cancellation', () => {
```

Comment:
Can we update the file name and the test description to include "-status-updates" or similar